src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue7396.cs

```diff
@@ -10,19 +10,21 @@ public Issue7396(TestDevice testDevice) : base(testDevice)
 	{
 	}
 
+	const string CreateTopTabButton = "CreateTopTabButton";
+	const string CreateBottomTabButton = "CreateBottomTabButton";
+	const string ChangeShellColorButton = "ChangeShellBackgroundColorButton";
+
 	public override string Issue => "Setting Shell.BackgroundColor overrides all colors of TabBar";
 
-	// TODO: Marked as ManualReview in original test, can we somehow automate this?
-	//[Test]
-	//[Category(UITestCategories.Shell)]
-	//public void BottomTabColorTest()
-	//{
-	//	//7396 Issue | Shell: Setting Shell.BackgroundColor overrides all colors of TabBar
-	//	App.WaitForElement(CreateBottomTabButton);
-	//	App.Tap(CreateBottomTabButton);
-	//	App.Tap(CreateBottomTabButton);
-	//	App.Tap(ChangeShellColorButton);
-	//	App.Screenshot("I should see a bottom tabbar icon");
-	//	Assert.Inconclusive("Check that bottom tabbar icon is visible");
-	//}
+	[Test]
+	[Category(UITestCategories.Shell)]
+	public void BottomTabColorTest()
+	{
+		//7396 Issue | Shell: Setting Shell.BackgroundColor overrides all colors of TabBar
+		App.WaitForElement(CreateBottomTabButton);
+		App.Tap(CreateBottomTabButton);
+		App.Tap(CreateBottomTabButton);
+		App.Tap(ChangeShellColorButton);
+		VerifyScreenshot();
```

Comment:
The Windows snapshots are pending and available in the latest build.
![image](https://github.com/user-attachments/assets/38e081a8-8637-4a63-9856-0435cb2ed85d)

Could you commit the images?

Reply:
https://dev.azure.com/xamarin/public/_build/results?buildId=129239&view=results

Reply:
Commited the images for windows UI tests.